dradis/dradis-qualys
# Add CVSSv3 fields to vuln uploader

Add `cvss3_base`, `cvss3_temporal`, and `cvss3_version` as available fields to the Qualys vuln uploader

### Testing Steps

1. Configure your Mappings Manager for Qualys Vuln Issues to import the following structure (or Mappings Manager equivalent!)
```
CVSSv2 Base: {{ qualys[element.cvss_base] }}
CVSSv2 Temporal: {{ qualys[element.cvss_temporal] }}
CVSSv3 Base: {{ qualys[element.cvss3_base] }}
CVSSv3 Temporal: {{ qualys[element.cvss3_temporal] }}
CVSSv3 Version: CVSSv3 Temporal: {{ qualys[element.cvss3_version] }}
```
2. Upload a sample Qualys file and confirm that the CVSSv3 data appears in the issues as expected.

### Copyright assignment
> I assign all rights, including copyright, to any future Dradis work by myself to Security Roots.

### Check List
